CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 Bugfix:
   * Fix java dependency on SLES 15 when building Puppet Platform 7
 
+## [2.5.6]
```

Comment:
I think this should be a `2.6.0` since we are adding a new interface.

Reply:
Which implies editing project.clj as well to match...

Reply:
Sure, updated